### ISSUE TITLE
Show multiple name validation messages

### ISSUE THREAD

**Issue by mgrbyte** [WormBase/names#287]
When a CGC or sequence-name is invalid, the error message returned for the HTTP 400 response is of the form:
```json
{"type":"validation-error","errors":[{"name":"cgc-name","value":" abc-12","reason":"Failed to match species regular expression pattern.","regexp":"^[a-z]{3,4}-[1-9]\\d*(\\.\\d+)?$"}],"message":"Missing or invalid name(s)."}
```
Can we display each error in `errors` below the top level message in the  error dialogue?

**Comment by sibyl229** [comment]
Should be possible. Is the API change available now?

**Comment by mgrbyte** [comment]
The multi message failures are available now, #288 adds more detail on the specific validation errors ( "reason" and "regexp" entries to the errors result maps (as shown above)
